packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/code-action/components/WorkflowEditActionServerlessFunction.tsx

```diff
@@ -289,6 +388,116 @@ export const WorkflowEditActionServerlessFunction = ({
 
   const testLogsTextAreaId = `${serverlessFunctionId}-test-logs`;
 
+  const getCodeEditorOptions = (includeLineNumbersConfig = true) => ({
```

Comment:
note that this will re-compute on every re-renders, so the result will also trigger a re-render itself.
Why not declaring the variable directly?

Reply:
Good point! Thanks